### ISSUE TITLE
Use Emphasis library for deep linking

### ISSUE THREAD

**Issue by englishm** [jasonphillips/laudato-si#1]
The NYTimes used to have a really great deep-linking feature.
It turns out that this was open sourced as a library called [Emphasis](https://github.com/NYTimes/Emphasis). (Thanks @donohoe!)

Maybe we could see if that will work here?

**Comment by jasonphillips** [comment]
I deployed an initial shot at this.  See this link:

http://jasonphillips.github.io/laudato-si/#p[OtoAie],h[EehIne,2,OtoAie,6]

However, I feel that more work is needed to make it fully useful. At present, one must be aware of the need to hit the SHIFT key twice to bring up the pg links; I should tie the existing linked paragraph numbers to this highlight functionality.

**Comment by donohoe** [comment]
The SHIFT key was specific to the NYTimes site but you could enable it by hover/tap/clicking a Paragraph to make that the 'active paragraph' etc.

**Comment by jasonphillips** [comment]
Sure; so I fixed that just now, making the paragraph numbers my trigger instead of a shift key, and also fiddled with improving the logic of sentence boundaries (not perfect yet, but prior to my changes any sentences ending in a footnote after the period were considered part of the sentence following). 

I'm nearly satisfied with the anchor and highlight functionality now. Although it could be a little more obvious and less hidden in my implementation.

**Comment by englishm** [comment]
This is looking great!

One minor issue I just noticed is that Emphasis seems to split sentences based on periods so you can sometimes get breaks like this: http://jasonphillips.github.io/laudato-si/#h[pg-2,5]

![laudato_si](https://cloud.githubusercontent.com/assets/1165411/8296607/75ca33ee-1920-11e5-8584-b9b1720c3d18.png)

That seems like it might be more effort than it's worth to fix though since I'm not sure how else sentences might easily be delineated. I can certainly tolerate this straightforward behavior.

**Comment by jasonphillips** [comment]
Good catch, I still need a few fixes on sentence logic. The problem is usually with other tags intervening; in that particular case, the Emphasis library would have handled "cf." correctly but my scripture reference tool added an extra tag in there for the popup.

**Comment by donohoe** [comment]
There is a hard-coded list of exceptions that you can extend. I'd love to hear ideas on a better way. Ultimately though someone can easily keep selecting so it does 'work', it could be better.

https://github.com/NYTimes/Emphasis/blob/master/js/emphasis-src.js#L440
